support-frontend/assets/components/subscriptionCheckouts/stage.jsx

```diff
@@ -24,6 +22,16 @@ type PropTypes = {|
   countryGroupId: CountryGroupId,
 |};
 
+type StagePropTypes = {
+  stage: Stage,
+  formSubmitted: boolean,
+  countryGroupId: CountryGroupId,
+  checkoutForm: Node,
+  thankYouContentPending: Node,
+  thankYouContentNotPending: Node,
```

Comment:
I think you could just call this `thankYouContent`, if you're worried it's not distinct enough you could call the other `pendingContent`.

Reply:
👍